_data/de/general.yml

```diff
@@ -11,3 +11,10 @@ nav:
   altJamulusIcon:    "Jamulus Icon. Link zur Wiki Startseite"
 wiki:
   titleAdd: "– Jamulus Wiki"
+blog:
+  titleAdd:          "– Jamulus Blog"
```

Comment:
Not sure we should call it a "blog" as that's too vague. See discussion here: https://github.com/jamulussoftware/jamuluswebsite/issues/56#issuecomment-735188557

Reply:
Agree.
But community section isn't good either.

Community posts is too long. Tips and Tricks from the community is even longer.

Reply:
What about 'Jamulus news' ?

Reply:
It doesn't really describe what it's for. It will be more something like a page where the community writes tips/tricks how to use Jamulus.

Reply:
'Jamulus community tips'n'tricks' ?

Reply:
Yeah, gilgongo also thought about that.

Reply:
Community Knowledge Base